SpatialGDK/Extras/schema/rpc_components.schema

```diff
@@ -26,3 +26,12 @@ component UnrealMulticastRPCEndpoint {
     id = 9987;
     event UnrealRPCPayload unreliable_multicast_rpc;
 }
+
+// Component that contains a list of RPCs to be executed
+// as a part of entity creation request
+component RPCsOnEntityCreation {
+	id = 9985;
+	list<UnrealRPCPayload> rpcs = 1;
+    command Void clear_rpcs(Void);
```

Comment:
Is this indented incorrectly or is GitHub weird

Reply:
Fixed.